app/models/pawn.rb

```diff
@@ -1,2 +1,19 @@
 class Pawn < Piece
+  def valid_move?(row_destination, col_destination)
+    start_row = self.is_black? ? 7 : 2
+    direction = self.is_black ? 1 : -1
+
+    row_distance = self.row - row_destination
+    col_distance = self.column - col_destination
+
+    if self.row == start_row && self.column == col_destination
+      return row_distance == 2 * direction || row_distance == 1 * direction
+    elsif row_distance == 1 * direction && col_distance == 1 * direction

```

Comment:
I think `col_distance == 1 * direction` will limit capture direction (left or right) based on the color of the piece.  You likely want `col_distance.abs == 1`

👍